base/applications/kbswitch/kbswitch.h

```diff
@@ -1,29 +1,17 @@
 #pragma once
 
-#include <stdarg.h>
-#include <windef.h>
-#include <winbase.h>
-#include <winuser.h>
-#include <winnls.h>
-#include <winreg.h>
-#include <wingdi.h>
+#include <windows.h>
```

Comment:
Try to use the separate headers as this was previously done.

Reply:
OK.